src/extensionApi.ts

```diff
@@ -278,7 +278,7 @@ export interface CompoundFilter {
   cqlFilter?: GenericCqlFilter;
   entityGroupFilter?: EntityGroupFilter;
   type: FilterType.COMPOUND_FILTER;
-  typeFilter: CatalogPageTypeFilter | null;
+  typeFilter?: CatalogPageTypeFilter | null;
```

Comment:
Alright so then that implies that the correct typing is 
```suggestion
  typeFilter?: CatalogPageTypeFilter;
```
right?

Reply:
Oh yes, sorry. I tested this and it works good - fixed.